membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -247,6 +259,10 @@ object SupporterRatePlanToAttributesMapper {
       ),
       "DEV" -> Map(
         List("guardian_patron") -> guardianPatronTransformer,
+        List(
+          "8ad09fc281de1ce70181de3b251736a4",
+          "8ad09fc281de1ce70181de3b28ee3783",
```

Comment:
shame we have these ids duplicated here, support-frontend repo and support-service-lambdas e.g. https://github.com/search?q=org%3Aguardian+2c92a0fb4edd70c8014edeaa4eae220a&type=code

Reply:
Agreed